packages/storybook/src/schematics/migrate-defaults-5-to-6/migrate-defaults-5-to-6.ts

```diff
@@ -0,0 +1,274 @@
+    context.logger.debug('adding .storybook folder to lib');
+    const workspaceJson = readWorkspace(tree);
+    const projects: string[] = Object.keys(workspaceJson.projects);
+    const projectsThatHaveStorybookConfiguration: {
+      name: string;
+      uiFramework: string;
+      configFolder: string;
+    }[] = projects.map((project: string) => {
+      if (
+        workspaceJson.projects[project].targets &&
+        workspaceJson.projects[project].targets.storybook
+      ) {
+        return {
+          name: project,
+          uiFramework:
+            workspaceJson.projects[project].targets.storybook.options
+              .uiFramework,
+          configFolder:
+            workspaceJson.projects[project].targets.storybook.options.config
+              .configFolder,
+        };
+      }
+      if (
+        workspaceJson.projects[project].architect &&
+        workspaceJson.projects[project].architect.storybook
+      ) {
+        return {
+          name: project,
+          uiFramework:
+            workspaceJson.projects[project].architect.storybook.options
+              .uiFramework,
+          configFolder:
+            workspaceJson.projects[project].architect.storybook.options.config
+              .configFolder,
+        };
+      }
+    });
+
+    return chain(
+      projectsThatHaveStorybookConfiguration
+        .filter(
+          (projectWithStorybook) =>
+            projectWithStorybook && projectWithStorybook.configFolder
+        )
+        .map((projectWithStorybook) => {
+          return chain([
+            migrateStorybookInstance(
+              keepOld,
+              projectWithStorybook.name,
+              projectWithStorybook.uiFramework,
+              projectWithStorybook.configFolder
+            ),
+          ]);
+        })
+    );
+  };
+}
+
+export function migrateStorybookInstance(
+  keepOld: boolean,
+  projectName: string,
+  uiFramework: string,
+  configFolder?: string
+): Rule {
+  return chain([
+    migrateProjectLevelStorybookInstance(
+      keepOld,
+      projectName,
+      uiFramework,
+      configFolder
+    ),
+  ]);
+}
+
+function upgradeStorybookPackagesInPackageJson(): Rule {
+  /**
+   * Should upgrade all @storybook/* packages in package.json
+   */
+  let needsInstall = false;
+
+  const maybeUpdateVersion = updateJsonInTree('package.json', (json) => {
+    json.dependencies = json.dependencies || {};
+    json.devDependencies = json.devDependencies || {};
+
+    const storybookPackages = [
+      '@storybook/angular',
+      '@storybook/react',
+      '@storybook/addon-knobs',
+    ];
+
+    storybookPackages.forEach((storybookPackageName) => {
+      if (json.dependencies[storybookPackageName]) {
+        const version = checkAndCleanWithSemver(
+          storybookPackageName,
+          json.dependencies[storybookPackageName]
+        );
+        if (lte(version, '6.0.0')) {
+          json.dependencies[storybookPackageName] = '^6.1.11';
```

Comment:
there's a `versions.ts` file in the Nrwl storybook package which has all the versions that are being used. I think we should reference that one. That way we can easily upgrade it in one place.